SQL/New_patches/2020-02-10-AddConsentGrouping.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE `consent_group` (
+  `ConsentGroupID` integer unsigned NOT NULL AUTO_INCREMENT,
+  `Name` varchar(255) NOT NULL,
+  `Label` varchar(255) NOT NULL,
+  CONSTRAINT `PK_consent_group` PRIMARY KEY (`ConsentGroupID`),
+  CONSTRAINT `UK_consent_group_Name` UNIQUE KEY `Name` (`Name`),
+  CONSTRAINT `UK_consent_group_Label` UNIQUE KEY `Label` (`Label`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+INSERT INTO `consent_group` (`ConsentGroupID`, `Name`, `Label`) VALUES ('1', 'main_consent', 'Main consent');
+
+ALTER TABLE `consent` ADD `ConsentGroupID` integer unsigned NOT NULL;
```

Comment:
@zaliqarosli have you tested this out? not sure what the behaviour would be since you are specifying `NOT NULL` but not giving a default. logically this would fail.

Reply:
I just tested it out, mariadb automatically makes the default 0 since no default was given. since you are adding a main consent category, I would suggest making it explicitly the default to avoid possible bugs